test/blackbox/common/BlackboxTestsDataRepresentationQos.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "BlackboxTests.hpp"
+
+#include <vector>
+#include <tuple>
+
+#include <gtest/gtest.h>
+
+#include <fastcdr/Cdr.h>
+
+#include "PubSubReader.hpp"
+#include "PubSubWriter.hpp"
```

Comment:
I think this is using the  wrong headers from `fastrtps_deprecated`.

Should be:
#include "../api/dds-pim/PubSubReader.hpp"
#include "../api/dds-pim/PubSubWriter.hpp"

Might be worth checking this for other tests.

Reply:
We have two different projects, one for each API.
The CMake files take care of adding the corresponding api folder to the list of include flags.